File: Listas de Exercícios (C)/C/Lista 1 - Sala/3.c
#include <stdio.h>

int main ()
{
    float V, r, h;
    
    printf("Volume de uma lata (cilindro)\n\n");
    
    printf("--Valores decimais devem ser digitados com (.)!--\n");
    printf("--Utilize a mesma unidade de medida!--\n");
    
    printf("Altura: "); scanf("%f", &h);
    printf("Raio da circunferência do círculo da base: "); scanf("%f", &r);
    
    V=h*r*3.14159;
    
    printf("\nVolume da lata: %.2f(medida)³\n", V);
}

File: Listas de Exercícios/Python/Python/Lista 1 - Sala/3.py
print("Volume de uma lata (cilindro)\n")

print("--Valores decimais devem ser digitados com (.)!--")

print("--utilize a mesma unidade de medida!--")
h=float(input("Altura: "))
r=float(input("Raio da circunferência do círculo da base: "))

V=h*r*3.14159; V="{:.2f}".format(V)

print(f"\nVolume da lata: {V}(medida)³\n")